client/app/tests/phpunit/Controller/CoDeputyControllerTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
```

Comment:
Why are we testing an insertion into the DB, do we not trust the DB to do a correct insertion? 
Do we have a tests on the endpoint `/selfregister/verifycodeputy` that would sufficiently cover this.

Reply:
There were no tests in place on either the client or API side, so I am creating a couple of tickets to get them created/refactored to be proper unit tests.